v2/internal/pkg/operator/filtered_collector.go

```diff
@@ -58,22 +56,15 @@ func (o *FilterCollector) OperatorImageCollector(ctx context.Context) (v2alpha1.
 			o.Log.Info("Collecting catalog %s", op.Catalog)
 		}
 		// prepare spinner
-		spinner := p.AddSpinner(
-			1, mpb.BarFillerMiddleware(spinners.PositionSpinnerLeft),
-			mpb.BarWidth(3),
-			mpb.PrependDecorators(
-				decor.OnComplete(spinners.EmptyDecorator(), emoji.SpinnerCheckMark),
-				decor.OnAbort(spinners.EmptyDecorator(), emoji.SpinnerCrossMark),
-			),
-			mpb.AppendDecorators(
-				decor.Name("("),
-				decor.Elapsed(decor.ET_STYLE_GO),
-				decor.Name(") Collecting catalog "+op.Catalog+" "),
-			),
-			mpb.BarFillerClearOnComplete(),
-			spinners.BarFillerClearOnAbort(),
-		)
-
+		p := mpb.New(mpb.ContainerOptional(mpb.WithOutput(io.Discard), !o.Opts.Global.IsTerminal))
+		spinner := spinners.AddSpinner(p, "Collecting catalog "+op.Catalog)
+		// CLID-47 double check that targetCatalog is valid
+		if op.TargetCatalog != "" && !v2alpha1.IsValidPathComponent(op.TargetCatalog) {
+			o.Log.Error(collectorPrefix+"invalid targetCatalog %s", op.TargetCatalog)
+			spinner.Abort(true)
+			spinner.Wait()
+			return v2alpha1.CollectorSchema{}, fmt.Errorf(collectorPrefix+"invalid targetCatalog %s", op.TargetCatalog)
+		}
```

Comment:
This is already part of `o.collectOperator`. Any reason for repeating it here?

Reply:
Done!